MergeCalendarsTogether.gs

```diff
@@ -11,17 +11,59 @@ const SYNC_DAYS_IN_FUTURE = 30;
 // Set this to "false" when your happy with the debug output!
 const DEBUG_ONLY = true;
 
+// Configure event summaries to ignore. These values are used with
+// RegExp.test() so when just a string literal, they act like a case-sensitive
+// "contains" check. If you want more control, use the line start (^) and/or
+// line end ($) regex symbols.
+const IGNORE_LIST_REGEXES = [
+  // 'Contains Match',
+  // '^Starts With Match',
+  // 'Ends With Match$',
+  // '^Some Exact Match$',
+  // '^Exact start.*Exact end$', // with anything in the middle
+]
+
+// should we copy event descriptions?
+const USER_INCLUDE_DESC = true;
+
 // ----------------------------------------------------------------------------
 // DO NOT TOUCH FROM HERE ON
 // ----------------------------------------------------------------------------
 
 const ENDPOINT_BASE = 'https://www.googleapis.com/calendar/v3/calendars';
 const MERGE_PREFIX = '🔄 ';
+const DESC_NOT_COPIED_MSG = '(description not copied)'
+// listed as first function so it's the default to run in the web UI
```

Comment:
that moment when you think "I should have really noticed that"

Reply:
:shrug: I started writing doco on how you should expect an error on the first run because it's "not the right function" until I realised it's just bad UX and it can be fixed :muscle:.